cypress.config.ts

```diff
@@ -0,0 +1,43 @@
+import { defineConfig } from "cypress";
```

Comment:
This file is mostly identical to the old config.json file, with a couple of differences